pyo3-build-config/src/impl_.rs

```diff
@@ -334,6 +354,15 @@ print("mingw", get_platform().startswith("mingw"))
         InterpreterConfig::from_reader(reader)
     }
 
+    /// Create an InterpreterConfig generated from pyo3-build-config using the "export-config"
+    /// feature
+    #[cfg(feature = "export-config")]
+    pub fn from_pyo3_export_config() -> Result<Self> {
```

Comment:
See also the hidden API `pyo3_build_config::get` (which probably should not be hidden), which looks like it achieves very similar result. Maybe we can combine the two?

Reply:
Yes this makes sense if it's okay make `pyo3_build_config::get` un-hidden. That function seems a bit magic so maybe the different cases should also be available to be called explicitly.